packages/esm-patient-registration-app/src/patient-registration/patient-registration-hooks.ts

```diff
@@ -108,7 +115,42 @@ export function useInitialFormValues(patientUuid: string): [FormValues, Dispatch
         setInitialFormValues(registration._patientRegistrationData.formValues);
       }
     })();
-  }, [initialFormValues, isLoadingPatientToEdit, patientToEdit, patientUuid]);
+  }, [
+    initialFormValues,
+    isLoadingPatientToEdit,
+    patientToEdit,
+    patientUuid,
+    fieldConfigurations.phone.personAttributeUuid,
+    isLocal,
+  ]);
+
+  useEffect(() => {
```

Comment:
This pattern feels off. This feels like it should be split into it's own hook or something an ultimately `useSWR()` if at all possible.